static/dbstructure.config.php

```diff
@@ -1292,6 +1292,33 @@
 			"guid" => ["guid(64)"],
 		]
 	],
+	"tag" => [
+		"comment" => "tags and mentions",
+		"fields" => [
+			"id" => ["type" => "int unsigned", "not null" => "1", "extra" => "auto_increment", "primary" => "1", "comment" => ""],
+			"name" => ["type" => "varchar(96)", "not null" => "1", "default" => "", "comment" => ""],
+			"url" => ["type" => "varbinary(255)", "not null" => "1", "default" => "", "comment" => ""]
+		],
+		"indexes" => [
+			"PRIMARY" => ["id"],
+			"type_name_url" => ["UNIQUE", "name", "url"],
+			"url" => ["url"]
+		]
+	],
+	"post-tag" => [
```

Comment:
Shouldn't it be `uri-tag` instead? It would be more consistent with other table naming since we don't have a `post` table.

Reply:
We don't have it yet. But I alreada have got a branch with the new item structure (where "post" is used)

Reply:
Oooooh, shiny!